Sources/Subscription/AccountManager.swift

```diff
@@ -168,6 +178,24 @@ public class AccountManager: AccountManaging {
         NotificationCenter.default.post(name: .accountDidSignOut, object: self, userInfo: nil)
     }
 
+    public func migrateAccessTokenToNewStore() throws {
+        // This migration is to prevent breaking internal tests and should be removed before launch
+        do {
+            if let newAccessToken = try accessTokenStorage.getAccessToken() {
+                throw MigrationError.noMigrationNeeded
```

Comment:
Not breaking functionality but this `throw` is caught down below and turned into `throw MigrationError.migrationFailed`